lib/promisepay_ex/client.ex

```diff
@@ -66,7 +66,15 @@ defmodule PromisepayEx.Client do
         }
       } ->
         {:ok, {401, headers, errors}}
-
+      {
```

Comment:
This is not needed as this function definition is for `GET` request 👉 https://github.com/psyfear/promisepay_ex/pull/24/files#diff-16be4c1424fc31c7d281df4ec9748181R48 ?